README.md

```
@@ -1,100 +1,105 @@
+### Pre-requisites
+- Python 3.10+
+- GNU Make
+- Compatible ESP32 device
+
+### Finding the BLE MAC address of your vehicle
+
+1. Copy and rename `secrets.yaml.example` to `secrets.yaml` and update it with your WiFi credentials (`wifi_ssid` and `wifi_password`) and vehicle VIN (`tesla_vin`).
+1. Enable the `tesla_ble_listener` package in `packages/base.yml` and build the firmware.
+1. Flash the firmware to your ESP32 device.
+1. Open the ESPHome logs in Home Assistant andto wake it up. watch for the "Found Tesla vehicle" message, which will contain the BLE MAC address of your vehicle.
+    > Note: The vehicle must be in range and awake for the BLE MAC address to be discovered. If the vehicle is not awake, open the Tesla app and run any command 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)